packages/desktop-client/src/components/banksync/AccountRow.tsx

```diff
@@ -62,14 +69,33 @@ export const AccountRow = memo(
           {account.bankName}
         </Cell>
 
-        <Cell
-          name="lastSync"
-          width={200}
-          plain
-          style={{ color: theme.tableText, padding: '10px' }}
-        >
-          {account.account_sync_source ? lastSync : ''}
-        </Cell>
+        {account.account_sync_source ? (
+          <Tooltip
+            placement="bottom start"
+            content={lastSyncDateTime}
+            style={{
+              ...styles.tooltip,
+            }}
+          >
+            <Cell
```

Comment:
Does it break anything to be omitting the Cell sometimes like this? Couldn't spot anything broken but not 100% sure

Reply:
It shouldn't, I believe the final cell is set to flex fill regardless so it's always right aligned.

Looks fine on the page in both cases.